pyanaconda/anaconda.py

```diff
@@ -221,18 +221,24 @@ def noninteractive_tui_mode(self):
 
     def _set_default_fstype(self, storage):
         fstype = None
+        boot_fstype = None
 
         # Get the default fstype from a kickstart file.
         if self.ksdata.autopart.autopart and self.ksdata.autopart.fstype:
             fstype = self.ksdata.autopart.fstype
+            boot_fstype = self.ksdata.autopart.fstype
```

Comment:
Was this the same before, eq. - that the fstype set from kickstart with "autopart --fstype" was also used for the bootloader ?